templates/games/card-game.html

```diff
@@ -1,28 +1,159 @@
 {% extends "base.html" %}
 {% load static %}
 {% block extra_css %}
-<link rel="stylesheet" href="{% static 'card-game.css' %}">
+    <link rel="stylesheet" href="{% static '/css/card-game.css' %}">
 {% endblock %}
 
 {% block content %}
 <!-- Content Goes here -->
-<h1>card game page</h1>
-
-
-
-
+<h1>Pharaoh`s Match</h1>
 
+<div class="container">
+    <div class="row">
+        <div class="col-sm-12 col-md-12 col-lg-12 col-xl-12 text-center">
+            <h3 class="game-header">Welcome to the world of the Pharaoh's and Ancient Egypt</h3>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-sm-12 col-md-12 col-lg-12 col-xl-12 text-center">
+            <div class="instructions">
+                <h4 class="instructions-header">how to play!</h4>
```

Comment:
What about capitalising _h_ in _how_ namely: **H**ow to play!

Reply:
I have used a CSS styling to make the h4 uppercase on the live site